src/Prima/Pyxis/Form/Select.elm

```diff
@@ -544,9 +544,16 @@ render model stateModel selectModel =
 
 renderSelect : model -> State -> Select model -> Html Msg
 renderSelect model ((State { choices }) as stateModel) selectModel =
+    let
+        options =
+            computeOptions selectModel
+
+        choicesWithPlaceholder =
+            SelectChoice "" options.placeholder :: choices
+    in
     Html.select
         (buildAttributes model stateModel selectModel)
-        (List.map (renderSelectChoice stateModel) choices)
+        (List.map (renderSelectChoice stateModel) choicesWithPlaceholder)
```

Comment:
Per evitarti il `let ... in` farei: 
```
renderSelect : model -> State -> Select model -> Html Msg
renderSelect model ((State { choices }) as stateModel) selectModel =
    Html.select
        (buildAttributes model stateModel selectModel)
        (options.placeholder  
           |> SelectChoice ""
           |> (::) choices 
           |> List.map (renderSelectChoice stateModel)
          )

Reply:
Figo! Ho dovuto mettere anche un flip prima del prepend.

Reply:
Giusto bravo :)